test/fixtures/flow/type-annotations/135/actual.js

```diff
@@ -0,0 +1,3 @@
+type A = {
+	...any,
```

Comment:
any is also not allowed here it seems, flow is throwing an error. Seems to also happen for all primitive types: `number, string, boolean, any, null, ...`